Makefile

```diff
@@ -89,6 +89,9 @@ op-devnet-allocs:
 benchmark:
 	go test -benchmem -run=^$ -bench . ./benchmark -test.parallel 4
 
+install-gotestsum:
```

Comment:
should just be called `deps`, and it should install all the requires dependencies/tooling that we need. My goal is to eventually use nix or mise for this, but in the meantime we can use this and add all the dependencies needed to run tests or develop locally.